packages/deployment/ansible/roles/cosmos-genesis/tasks/main.yml

```diff
@@ -88,7 +88,7 @@
   become_user: "{{ service }}"
   shell: "\
     {{ service }} gentx --keyring-dir=/home/{{ service }}/.ag-cosmos-helper \
-      --keyring-backend=test --amount={{ STAKER_AMOUNT }} \
+      --keyring-backend=test {{ STAKER_AMOUNT }} \
```

Comment:
what a pain to change this API convention all over the place. Part of me is inclined to go find their justification for this backward-incompatible change. But I'll just make my peace with it, for now.